test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -74,6 +81,25 @@ describe('IexecPoco1', () => {
     let [randomAddress, randomSignature]: string[] = [];
     let randomContract: OwnableMock;
     let erc1271MockContract: ERC1271Mock;
+    let iexecPocoManageOrderTx: () => Promise<ContractTransaction>;
+    let providerAddress: string;
+    let order:
+        | IexecLibOrders_v5.AppOrderStruct
+        | IexecLibOrders_v5.DatasetOrderStruct
+        | IexecLibOrders_v5.WorkerpoolOrderStruct
+        | IexecLibOrders_v5.RequestOrderStruct;
+    let orderHash: string;
+    let orderManagement: {
+        [key: string]: {
+            iexecPocoManageOrderTx: () => Promise<ContractTransaction>;
```

Comment:
Rename both `iexecPocoManageOrderTx` l84 & 94 & etc to `iexecPocoSignManageOrder`